doc/classes/VisualShaderNodeFloatFunc.xml

```diff
@@ -1,81 +1,116 @@
 		</constant>
 		<constant name="FUNC_SIGN" value="13" enum="Function">
+			Extracts the sign of the parameter. Translates to [code]sign(x)[/code] in Godot Shader Language.
 		</constant>
 		<constant name="FUNC_FLOOR" value="14" enum="Function">
+			Finds the nearest integer less than or equal to the parameter. Translates to [code]floor(x)[/code] in Godot Shader Language.
 		</constant>
 		<constant name="FUNC_ROUND" value="15" enum="Function">
+			Finds the nearest integer to the parameter. Translates to [code]round(x)[/code] in Godot Shader Language.
 		</constant>
 		<constant name="FUNC_CEIL" value="16" enum="Function">
+			Finds the nearest integer that is greater than or equal to the parameter. Translates to [code]ceil(x)[/code] in Godot Shader Language.
 		</constant>
 		<constant name="FUNC_FRAC" value="17" enum="Function">
+			Computes the fractional part of the argument. Translates to [code]fract(x)[/code] in Godot Shader Language.
 		</constant>
 		<constant name="FUNC_SATURATE" value="18" enum="Function">
+			Clamps the value between [code]0.0[/code] and [code]1.0[/code] by [code]min(max(x, 0.0), 1.0)[/code] formula.
 		</constant>
 		<constant name="FUNC_NEGATE" value="19" enum="Function">
+			Negates the [code]x[/code] by [code]-(x)[/code] formula.
 		</constant>
 		<constant name="FUNC_ACOSH" value="20" enum="Function">
+			Returns the arc-hyperbolic-cosine of the parameter. Translates to [code]acosh(x)[/code] in Godot Shader Language.
 		</constant>
 		<constant name="FUNC_ASINH" value="21" enum="Function">
+			Returns the arc-hyperbolic-sine of the parameter. Translates to [code]asinh(x)[/code] in Godot Shader Language.
 		</constant>
 		<constant name="FUNC_ATANH" value="22" enum="Function">
+			Returns the arc-hyperbolic-tangent of the parameter. Translates to [code]atanh(x)[/code] in Godot Shader Language.
 		</constant>
 		<constant name="FUNC_DEGREES" value="23" enum="Function">
+			Convert a quantity in radians to degrees. Translates to [code]degrees(x)[/code] in Godot Shader Language.
 		</constant>
 		<constant name="FUNC_EXP2" value="24" enum="Function">
+			Returns 2 raised by the power of the parameter. Translates to [code]exp2(x)[/code] in Godot Shader Language.
 		</constant>
 		<constant name="FUNC_INVERSE_SQRT" value="25" enum="Function">
+			Returns the inverse of the square root of the parameter. Translates to [code]inversesqrt(x)[/code] in Godot Shader Language.
 		</constant>
 		<constant name="FUNC_LOG2" value="26" enum="Function">
+			Returns the base 2 logarithm of the parameter. Translates to [code]log2(x)[/code] in Godot Shader Language.
 		</constant>
 		<constant name="FUNC_RADIANS" value="27" enum="Function">
+			Convert a quantity in degrees to radians. Translates to [code]radians(x)[/code] in Godot Shader Language.
 		</constant>
 		<constant name="FUNC_RECIPROCAL" value="28" enum="Function">
+			Finds reciprocal value of dividing 1 on [code]x[/code] by [code]1 / x[/code] formula.
```

Comment:
Finds reciprocal value of dividing 1 _by_ [code]x[/code] (i.e. [code]1 / x[/code]).